docs/sources/reference/components/beyla.ebpf.md

```diff
@@ -120,14 +120,23 @@ It contains the following blocks:
 ### services block
 
 In some scenarios, Beyla will instrument a wide variety of services, such as a Kubernetes DaemonSet that instruments all the services in a node.
-This block allows you to filter the services to instrument based on their metadata.
+This block allows you to filter the services to instrument based on their metadata. If you specify other selectors in the same services entry,
+the instrumented processes need to match all the selector properties.
 
 Name         | Type     | Description                                                                     | Default | Required
 -------------|----------|---------------------------------------------------------------------------------|---------|---------
 `name `      | `string` | The name of the service to match.                                               |         | no
 `namespace`  | `string` | The namespace of the service to match.                                          |         | no
 `open_ports` | `string` | The port of the running service for Beyla automatically instrumented with eBPF. |         | no
 `exe_path`   | `string` | The path of the running service for Beyla automatically instrumented with eBPF. |         | no
+`k8s_namespace` | `string` | Regular expression of Kubernetes Namespaces to match.                          |         | no
+`k8s_pod_name`  | `string` | Regular expression of Kubernetes Pods to match.                           |         | no
```

Comment:
Are these mutually exclusive with setting name/namespace?

Reply:
From what can I read in Beyla docs, no, it has to match everything defined in the `services` section. I added a note [here](https://github.com/grafana/alloy/pull/928/files#diff-01b6ebc9a981ba6d65085b056aa9b981225c215a8e7b7bdb8cc30e86ecc4c061R123-R124). Let me know if it's enough or needs more clarification.